builtin/rebase.c

```diff
@@ -695,7 +715,7 @@ int cmd_rebase(int argc, const char **argv, const char *prefix)
 	trace_repo_setup(prefix);
 	setup_work_tree();
 
-	git_config(git_default_config, NULL);
+	git_config(rebase_config, &options);
```

Comment:
I would also note in the commit message that this is the first patch introducing handling for `rebase.*` config settings, and the others will follow. In the commit history, these commits should be huddling together, to make for a better story.